assets/js/common_utils.js

```diff
@@ -119,33 +119,28 @@ export const getTime = () => {
 
 const KNOWN_COMPATIBLE_ORT_VERSION = {
     "stable-diffusion-1.5": {
-        dev: "1.20.0-dev.20240919-bd60add8ce",
-        stable: "1.20.0",
-        test: "test",
+        dev: "1.21.0-dev.20241109-d3ad76b2cf",
+        stable: "",
+        test: "",
     },
     "sd-turbo": {
-        dev: "1.20.0-dev.20240919-bd60add8ce",
-        stable: "1.20.0",
-        test: "test",
+        dev: "1.21.0-dev.20241109-d3ad76b2cf",
+        stable: "",
```

Comment:
@fdwr Since no demos currently use the stable version, we should only add version information for the ones actually in use and leave the others empty. What do you think?

Reply:
Ok.